### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/6898f4f6dd7104000853e9e6?deploy_diagnostics=true)**

### ISSUE THREAD

**Issue by Jobayer071** [stackblitz/docs#248]
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/6898f4f6dd7104000853e9e6?deploy_diagnostics=true)**

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 989675e614c36588379f6759af1705409920a116 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/stackblitz-docs/deploys/6898f4f6dd7104000853e9e6 |

_Originally posted by @netlify[bot] in https://github.com/stackblitz/docs/issues/240#issuecomment-3172848926_
